### PR TITLE
Bump swagger client to version 1.0.48-rc9

### PR DESCRIPTION
**Pull request by gaelL**

### CODE REVIEW

**Review by talset — APPROVED** [review]

File: printer/table/printer.go [hunk 1]
```diff
@@ -33,6 +34,20 @@ func entryFromStruct(obj reflect.Value, h []string) []string {
 			values = append(values, value.String())
 		case reflect.Uint32:
 			values = append(values, strconv.FormatInt(int64(value.Uint()), 10))
+		case reflect.Slice:
+			// Render Slice of strings
+			typ := value.Type().Elem()
+			if typ == reflect.TypeOf(string("")) {
```
Comment (tormath1) [inline printer/table/printer.go:40]: Is it necessary ? We already know that `value` is a slice, we can directly iterate on it. :thinking:

Comment (gaelL) [reply printer/table/printer.go:40]: We can but we don't know if it's a slice of string, int or whatever right ?

Comment (tormath1) [reply printer/table/printer.go:40]: This is what we're doing later: 
https://github.com/cycloidio/cycloid-cli/blob/add905af0aa41c680a6180a8bfa521434e350594/printer/table/printer.go#L43-L45

No ?

Comment (gaelL) [reply printer/table/printer.go:40]: Indeed, I forgot to remove this one.

**Comment by gaelL** [comment]
comments applied

**Review by tormath1 — APPROVED** [review]

**Pull request merged**